browser/components/ColorPicker.styl

```diff
@@ -0,0 +1,37 @@
+.colorPicker
+  position fixed
+  z-index 2
+  display flex
+  flex-direction column
+
+.cover
+  position fixed
+  top 0
+  right 0
+  bottom 0
+  left 0
+
+.footer
+  display flex
+  justify-content center
+  z-index 2
+  align-items center
+  & > button + button
+    margin-left 10px
+
+.btn-cancel,
+.btn-confirm,
+.btn-reset
+  vertical-align middle
+  height 25px
+  margin-top 2.5px
+  colorDefaultButton()
+  border-radius 2px
+  border $ui-border
```

Comment:
It looks better with `border: none`. Also when I hover the button, its' background is transparent
![ab](https://user-images.githubusercontent.com/12984316/50546942-4aef4080-0c63-11e9-9600-f45dfd0e642a.gif)

Reply:
@ZeroX-DG Updated